lib/reactive_table.html

```diff
@@ -69,13 +69,80 @@
+                  <td class="{{getCellClass ..}}">
+                    {{#if tmpl}}
+                      {{#with ..}}
+                        {{#if ../expandButton}}
+                          {{#with extendExpandHelpers}}
+                            {{> ../../tmpl}}
+                          {{/with}}
+                        {{else}}
+                          {{> ../tmpl}}
+                        {{/if}}
+                      {{/with}}
+                    {{else}}
+                      {{#if expandButton}}
+                        {{#if ../../useFontAwesome}}
+                          <i class="{{../../childrenExpandIconClass}} fa fa-plus-square-o" data-id="{{../_id}}"></i>
+                        {{else}}
+                          <label class="{{../../childrenExpandIconClass}}" data-id="{{../_id}}">&#10133;</label>
+                        {{/if}}
+                      {{/if}}
+                      {{getField ..}}
+                    {{/if}}
+                   </td>
                 {{/if}}
               {{/each}}
             </tr>
+            {{#if ../children}}
+              {{#if isVisibleChild}}
+                <tr class="reactive-table-children-container">
+                  <td colspan="{{../fields.length}}">
+                    {{#if ../children/fields}}
+                      <table class="table reactive-table-children">
+                        <thead>
+                        <tr>
+                          {{#each ../children.fields}}
+                            <th class="{{getHeaderClass}}">
+                              {{label}}
+                            </th>
+                          {{/each}}
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {{#each getChildrenRows}}
+                          <tr>
+                            {{#each ../../children.fields}}
+                              <td class="{{getCellClass ..}}">
+                                {{#if tmpl}}
+                                  {{#with ..}}
+                                    {{> ../tmpl}}
+                                  {{/with}}
+                                {{else}}
+                                  {{getField ..}}
+                                {{/if}}
+                              </td>
+                            {{/each}}
+                          </tr>
+                        {{/each}}
+                        </tbody>
+                      </table>
+                    {{else}}
+                      {{#if ../children/tmpl}}
+                        {{#with extendExpandHelpers}}
+                          {{> ../../children/tmpl}}
+                        {{/with}}
+                      {{else}}
+                        <span class="error">if children field exists, you must have defined either tmpl or fields</span>
```

Comment:
This is kind of a nitpick, but I don't think we should have any errors that could accidentally get displayed to an end user if there's a bug in someone's code. `setup` in reactive-table.js can check all the children and log an error to the console if there are problems.